### DISCUSSION TITLE
Here is the complete TLS add-on (HTTS, StartTLS, SMTPS)

### DISCUSSION THREAD

**Discussion by zdeneksvarc** [postalserver/postal#2140] [Help with using Postal]
Based on Github discussions, I created a repo around docker-compose.yml, which adds HTTPS, StartTLS (587) mail submission and implicit TLS (465) mail submission to the basic Postal installation. Tested on Postal  2.1.1

- https://github.com/zdeneksvarc/postal-tls

I'd appreciate any further thoughts 🙋🏻‍♂️

**Comment by jshapiro2** [comment]
@zdeneksvarc please reach out to me on skype @shap906

**Reply by SimonBiggs** [reply]
Yup, it would be great if the default install defined within the install docs pulled in the Caddy certificates for SMTP TLS encryption.

See the steps I underwent here based on the caddy install:

https://github.com/postalserver/postal/issues/98#issuecomment-1486144615

Where I was able to use the Caddy outputs to have the SMTP logins be secured with TLS
